google-cloud-bigtable/lib/google/cloud/bigtable/service.rb

```diff
@@ -226,18 +221,16 @@ def delete_instance instance_id
           end
         end
 
-        # Create a cluster within an instance.
+        # Creates a cluster within an instance.
```

Comment:
Good question. I started by thinking that the headings were usually imperatives and the method descriptions were in present tense, but I don't think that's consistent throughout, and I wound up just trying to be consistent within a file.

Reply:
I think google-cloud-bigtable mixes both forms. In all other google-cloud-ruby libraries, we use present tense. (This is standard in the Ruby community, see Ruby Core docs and [Rails doc guidelines](https://guides.rubyonrails.org/api_documentation_guidelines.html#wording).) It would be great to convert all of google-cloud-bigtable docs to present tense, but I didn't want to assume that was in scope for this PR. I merely wondered why you would bother changing a few usages.

Reply:
OK, in service.rb I've made them all present tense, and I'll follow that rule moving forward but not go back to files that I've already edited, in order to avoid scope creep. Thanks for raising this.